refinery/core/test_views.py

```diff
@@ -934,3 +935,78 @@ def test_get_workflow_graph(self):
             uuid=self.workflow.uuid
         )
         self.assertEqual(get_response.content, self.mock_workflow_graph)
+
+
+class UserProfileApiV2Tests(APIV2TestCase):
+    def setUp(self, **kwargs):
+        super(UserProfileApiV2Tests, self).setUp(
+            api_base_name="user_profiles/",
+            view=UserProfileViewSet.as_view()
+        )
+        self.user_lm = User.objects.create_user('lab_member',
+                                                'member@fake.com',
+                                                self.password)
+        self.lab_group = ExtendedGroup.objects.create(name="Lab Group")
+        self.non_lab_group = ExtendedGroup.objects.create(name="Test Group")
+        self.lab_group.group_ptr.user_set.add(self.user_lm)
```

Comment:
I think you can access `user_set` directly without going through `group_ptr`:
`self.lab_group.user_set...`